### PR TITLE
travis ci -> github actions

### PR DESCRIPTION
**Pull request by fgregg**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:9.6
```
Comment (hancush) [inline .github/workflows/main.yml:18]: We use Postgres 10 in deployment. Can you update here?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      - master
+      - deploy
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
```
Comment (hancush) [inline .github/workflows/main.yml:15]: Our prod server is on Ubuntu 16. 🙀 I don't think this is [an available runtime for GH Actions anymore](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners). Any point in using the oldest OS that is available (Ubuntu 18)?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        ports:
+          - 6379:6379
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.6'
```
Comment (hancush) [inline .github/workflows/main.yml:40]: Yikes, we're still on 3.5. Update?

**Review by hancush — CHANGES_REQUESTED** [review]
Thanks, @fgregg! A few things inline for version compatibility with our image and deployment.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -41,6 +41,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install setuptools_scm
```
Comment (hancush) [inline .github/workflows/main.yml:44]: Do we need to add this step to the deploy script, as well? (Could failure to do so result in failed prod deployment?)

Comment (hancush) [reply .github/workflows/main.yml:44]: Retroactively tagging @fgregg

Comment (fgregg) [reply .github/workflows/main.yml:44]: it wasn't present before, so i would wait to do that.

**Comment by fgregg** [comment]
I'm going to go ahead and merge this. if there is a deployment failure, i'll update the script.

**Pull request merged**